### DISCUSSION TITLE
How do I remove my old VM?

### DISCUSSION THREAD

**Discussion by WB3GitHub81542** [Parallels/parallels-vscode-extension#108] [Q&A]
I moved from MacBook Pro using Windows 10 to MacBook Pro M2 using Windows 11, how to I remove the old VM?

**Comment by cjlapao** [comment]
@WB3GitHub81542 The vscode should only list the local vms, so if this is a different hardware and the vms were not moved this should not list it.

Can you also see this VM in the Parallels Control Center?
